### PR TITLE
https://issues.redhat.com/browse/ACM-13319

### PR DESCRIPTION
**Pull request by jc-berger**
see jira: https://issues.redhat.com/browse/ACM-13319

### CODE REVIEW

**Comment by swopebe** [comment]
requesting hangout to go over the steps surrounding this, what and how to figure out what to code and some of the wording.

File: gitops/gitops_service_account_argo_cd.adoc [hunk 1]
```diff
@@ -112,6 +112,7 @@ The GitOpsCluster resource uses placement to import selected managed clusters in
 ----
 ---
```
Comment (dockerymick) [inline gitops/gitops_service_account_argo_cd.adoc]:
```suggestion
```

**Review by dockerymick — APPROVED** [review]
Remove the extra lines, otherwise this looks good

**Comment by swopebe** [comment]
Live review of necessary changes and things to look for with steps and YAML.

**Review by swopebe — APPROVED** [review]

**Pull request merged**